infrastructure/helm-chart/charts/apps/charts/airy-config/templates/kafka.yaml

```diff
@@ -5,3 +5,4 @@ metadata:
 data:
   KAFKA_BROKERS: {{ .Values.kafka.brokers }}
   KAFKA_SCHEMA_REGISTRY_URL: {{ .Values.kafka.schemaRegistryUrl }}
+  KAFKA_COMMIT_INTERVAL_MS: {{ .Values.kafka.commitInterval }}
```

Comment:
You could put the default here like `.Values.kafka.commitInterval | default 30000` 
so you only have it in one place